app/ota_metadata.py

```diff
@@ -319,6 +345,7 @@ class RegularInf:
     example: 0644,1000,1000,1,0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef,'path/to/file',1234,12345678
 
     NOTE: size and inode sections are both optional, if inode exists, size must exist.
+    NOTE 2: path should always be canonical!
```

Comment:
What do you mean by this line? For the regulars.txt, path is always absolute. If function argument `path` should be canonical, why don't you Path.resolve()?

Reply:
Oh this line is kind of misleading, what I mean here is that path in RegularInf should always relative to `/`, not to the standby slot mount point. I will rephrase this comment.